packages/basepaint/src/Basepaint.test.ts

```diff
@@ -0,0 +1,58 @@
+import { GreaterThanOrEqual, apply } from '@rabbitholegg/questdk/filter'
+import { describe, expect, test } from 'vitest'
+import { mint } from './Basepaint'
+import { CONTRACT_ADDRESS, MINT_ABI } from './constants'
+
+// Replace *project* with the name of the project
+describe('Given the basepaint plugin', () => {
+  describe('When handling the mint filter', () => {
+    test('should return a valid action filter', async () => {
+      const filter = await mint({
+        address: '0xba5e05cb26b78eda3a2f8e3b3814726305dcac83',
+        tokenId: 69,
+        quantity: GreaterThanOrEqual(1),
+      })
+      expect(filter).to.deep.equal({
+        to: CONTRACT_ADDRESS,
+        chainId: 8453,
+        input: {
+          $abi: MINT_ABI,
+          day: 69,
+          count: {
+            $gte: '1',
+          },
+        },
+      })
+    })
+
+    test('should pass filter with valid transactions', async () => {
```

Comment:
Can we move the transaction object to the constants file and add a second transaction that it fails for?

Reply:
Done! I added additional tests to validate that the filter correctly rejects transactions with an incorrect tokenId or quantity.